### PR TITLE
last batch of changes as disscussed in the last meeting

### PR DESCRIPTION
**Pull request by golnazads**

### CODE REVIEW

**Comment by seasidesparrow** [comment]
Are the publisher-source scores being assigned somewhere, or does that happen in another service?  From CSG email:
```
spires                  1.05
publisher              1.1
author-submitted 1.2
ads-curated         1.3
```

**Comment by golnazads** [comment]
They are in the oracle database.

**Comment by golnazads** [comment]
yes.

File: adsdocmatch/oracle_util.py [hunk 1]
```diff
@@ -2,21 +2,24 @@
...
+config = load_config(proj_home=proj_home)
 
-logger = setup_logging("docmatching", level=conf.get("LOGGING_LEVEL", "WARN"), proj_home=proj_home, attach_stdout=conf.get("LOG_STDOUT", "FALSE"))
+logger = setup_logging("docmatching", level=config.get("LOGGING_LEVEL", "WARN"), proj_home=proj_home, attach_stdout=config.get("LOG_STDOUT", "FALSE"))
 
 
 class OracleUtil():
 
-    COLLABORATION_PAT = re.compile(r"(?P<collaboration>[(\[]*[A-Za-z\s\-\/]+\s[Cc]ollaboration[s]?\s*[A-Z\.]*[\s.,)\]]+)")
+    COLLABORATION_PAT = re.compile(r"(?P<collaboration>[(\[]*[A-Za-z\s\-\/]+\s[Cc]ollaboration[s]?\s*[A-Z\.]*)(?:[\s.,;)\]]+|$)")
```
[the remark this answers is not in the document]
Comment (seasidesparrow) [reply adsdocmatch/oracle_util.py:22]: I was looking for something that (I think?) fails with the original regex but passes the with the updated one.  This string does that: 'Copernicus, Nicholas; Galilei, Galileo; Lijiao, Liu; the ALICE Collaboration'. I don't think it's required, but would provide some clarity as to what the test is doing.

**Review by seasidesparrow — APPROVED** [review]
Looks good, including the changes to utils.py / Spreadsheet handling.  Minor comment regarding the modified regex in oracle for your consideration, otherwise no issues.

File: adsdocmatch/oracle_util.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (golnazads) [reply adsdocmatch/oracle_util.py:22]: Cool. thank you. Going to add the comment, and then if you would please merge, release, and have it deployed. I would appreciate it, if you also run it in your test container testing each command and making sure all is correct, before pushing it forward. Oracle changes are in dev, so you should be able to verify the new commands. Thank you again Matt.

[the remark this answers is not in the document]
Comment (golnazads) [reply adsdocmatch/oracle_util.py:23]: committed the change.

**Pull request merged**